docs/CONTRIBUTING.md

```diff
@@ -222,3 +222,39 @@ extensions.lmem@gmail.com
 
 To generate your own tokens and deploy from your local environment :
 https://github.com/DrewML/chrome-webstore-upload/blob/master/How%20to%20generate%20Google%20API%20keys.md
+
+### Firefox Addons
+
+[Developer Hub](https://addons.mozilla.org/en-US/developers/addons)
+| account: infrastructure@lmem.net
+
+In order for the Mozilla review to complete successfully, please the following steps :
+
+- Use [SemaphoreCI](https://semaphoreci.com/lmem/extension/) to deploy to `Firefox production`
+- [Download](https://github.com/dis-moi/extension/releases) the source code archive of the version you want to deploy to production extension (`Source code (tar.gz)`)
+- Create a `buildDisMoi.sh` file on your machine with the following content :
```

Comment:
Could we ease the process and prepare at least part of this script ?
Maybe embed a part of it in the `package.json` or a dedicated script in `bin` dir.